Zotero/Scenes/Detail/PDF/Views/Annotation View/AnnotationView.swift

```diff
@@ -232,11 +232,12 @@ final class AnnotationView: UIView {
         })
         self.commentTextView.textObservable
             .debounce(.milliseconds(500), scheduler: MainScheduler.instance)
-            .subscribe(onNext: { text, needsHeightReload in
-                self.actionPublisher.on(.next(.setComment(text)))
+            .subscribe(onNext: { [weak self] data in
+                guard let (text, needsHeightReload) = data else { return }
```

Comment:
Add `let self` to `guard` and remove `self?`.

Reply:
Check https://github.com/zotero/zotero-ios/pull/831/commits/712a9acd8014d8279613bf2f9f1670d42f1cd8e6
The `let self = self` style is necessary, otherwise the compiler complains. Maybe they forgot to update some compiler logic for function builders?